src/main/java/gregtech/api/metatileentity/multiblock/IProgressBarMultiblock.java

```diff
@@ -24,14 +32,26 @@ default int getNumProgressBars() {
     double getFillPercentage(int index);
 
     /** Textures for the progress bar(s). */
+    @Deprecated
     default TextureArea getProgressBarTexture(int index) {
         return GuiTextures.PROGRESS_BAR_MULTI_ENERGY_YELLOW;
     }
 
+    default UITexture getProgressBarTexture2(int index) {
+        return GTGuiTextures.PROGRESS_BAR_MULTI_ENERGY_YELLOW;
+    }
+
+    default int getProgressBarTextureHeight(int index) {
+        return 18;
+    }
+
     /**
      * Add hover text to your progress bar(s).
      *
      * @param index The index, 0, 1, or 2, of your progress bar. Only relevant if you have multiple bars.
      */
+    @Deprecated
     default void addBarHoverText(List<ITextComponent> hoverList, int index) {}
+
+    default void addBarHoverText2(@NotNull Tooltip tooltip, int index) {}
```

Comment:
Why bother adding Mui2 methods to this class? If they have to change the method already, it would be better to just directly change to the MUI2 implementation in `ProgressBarMultiblock`.

Reply:
these were left over when tech started work, i think they can be removed now. in fact, nothing internally uses this interface anymore, should i remove it?

Reply:
I would say don't remove the interface, since we want addon multiblocks to still be able to work with MUI for a bit. But you can remove the MUI2 specific things from this interface, since the new interface exists now.